judge/views/contests.py

```diff
@@ -104,6 +107,9 @@ def get_context_data(self, **kwargs):
         context['future_contests'] = future
         context['now'] = self._now
         context['first_page_href'] = '.'
+        context.update(self.get_sort_context())
+        context.update(self.get_sort_paginate_context())
+
```

Comment:
This (newline) occasionally happens in the code base, but we're somewhat inconsistent and it shouldn't be added if it wasn't there before.